Firebase.config.js

```diff
@@ -0,0 +1,12 @@
+import { initializeApp } from "firebase/app";
+
+const firebaseConfig = {
+  apiKey: "AIzaSyDzawW7qnt7M-SjlPr7_Gg22GAhJNz_0wE",
+  authDomain: "damage-control-ba92f.firebaseapp.com",
+  projectId: "damage-control-ba92f",
+  storageBucket: "damage-control-ba92f.appspot.com",
+  messagingSenderId: "127685083270",
+  appId: "1:127685083270:web:58bf57eb6d5dfaf3bc1107",
+};
+
+export const app = initializeApp(firebaseConfig);
```

Comment:
File must be a secret, use `dotenv` module or something like that to not expose secret tokens

Reply:
Thanks for the recommendation, it has just been implemented.